server/build_event_protocol/accumulator/accumulator.go

```diff
@@ -6,6 +6,8 @@ import (
 
 	"github.com/buildbuddy-io/buildbuddy/proto/build_event_stream"
 	"github.com/buildbuddy-io/buildbuddy/proto/command_line"
+
+	gitutil "github.com/buildbuddy-io/buildbuddy/server/util/git"
```

Comment:
we should name this file + package gitutil too

Reply:
OK, let me send that as a follow-up PR so it's an easy-LGTM